client/src/Views/Landing/Login.js

```diff
@@ -17,9 +29,32 @@ class Login extends Component {
     });
   };
 
-  onClick = async e => {};
+  onSubmit = async () => {
+    const fields = { ...this.state };
+
+    try {
+      const res = await login(fields);
+      localStorage.setItem("token", `Bearer ${res.data.token}`);
```

Comment:
I feel like the localStorage logic should be put into a separate file in the Utils folder, so we can easily modify the auth logic when we need to (in case we want to change it)

Reply:
In my user context PR I have moved this to the withAuthentication.js file. Login now just calls context.setToken()